SubscribeIT/ngDesk
# 23 knowladgebase section, category and articles api integration

**Closes #23

**Test case 1 :**
Name :- Category page

1. Login to ngdesk.
2. Goto -> knowledge base.
3. Hit on New button .
4. Click on  category, fill the required fields and hit the save button.
5. Click Manage Category button > click on edit > fill the required fields and hit the save button.
6. Observe the data updated.
7. Click Manage Category button > click on edit > click on delete category button.
8. Observe category was deleted.
9. Observe No errors.

**Test case 2 :**
Name :- Section 

1. Login to ngdesk.
2. Goto -> knowledge base.
3. Hit on New button .
4. Click on Section, fill the required fields and hit the save button.
5. Click Manage Section button > click on edit > fill the required fields and hit the save button.
6. Observe the data updated.
7. Click Manage Section button > click on edit > click on delete section button.
8. Observe section was deleted.
9. Observe No errors.

**Test case 3 :**
Name :- Article 

1. Login to ngdesk.
2. Goto -> knowledge base.
3. Hit on New button .
4. Click on article, fill the required fields and hit the save button.
5. Click Manage article button > click on edit > fill the required fields and hit the save button.
6. Observe the data updated.
7. Click Manage article button > click on edit > click on delete article button.
8. Observe article was deleted.
9. Observe No errors.

**Test case 4 :**
Name :- Manage Article 

1. Login to ngdesk.
2. Goto -> knowledge base.
3. Hit on Manage Article button .
4. Click on any article, fill the required fields and hit the save button.
5. observe the data updated.
6. Click on any article>click on delete article button.
7. Observe article was deleted.
8. Observe No errors.


**Test case 5 :**
Name :- knowledge base

1. Login to ngdesk.
2. Goto -> knowledge base.
3. Hit on New button .
4. Click on  category, fill the required fields and hit the save button.
5. Hit on Manage Article button> select add > click on section.
6. Observe the create section page > fill the required page and hit the save button.
7. Click Manage section button > select add > click on article.
8. Observe the article page > fill the required page and hit the save button.
9. Observe the edit article page > fill the required page and hit the save button.
10. observe the data updated.
11. Observe no errors.

**Screenshots:**


![Screenshot from 2021-09-16 08-06-00](https://user-images.githubusercontent.com/89505134/133542254-1a3fc8b8-166d-4516-a18c-a66b4a43a7e6.png)
![Screenshot from 2021-09-16 08-06-10](https://user-images.githubusercontent.com/89505134/133542260-8319ff94-157a-458f-927d-5d0c929121a5.png)
![Screenshot from 2021-09-16 08-06-39](https://user-images.githubusercontent.com/89505134/133542263-1d2cf15a-ca56-4c69-8eb7-0c620bad65b1.png)
![Screenshot from 2021-09-16 08-06-43](https://user-images.githubusercontent.com/89505134/133542265-7d43c710-183e-4ab1-86df-73aa0827b039.png)
![Screenshot from 2021-09-16 08-06-52](https://user-images.githubusercontent.com/89505134/133542266-092d9815-c065-4735-86cd-a22304098bdb.png)
![Screenshot from 2021-09-16 08-07-08](https://user-images.githubusercontent.com/89505134/133542267-d4dca526-1a9a-440e-8c60-e0ce8048727f.png)
![Screenshot from 2021-09-16 08-07-17](https://user-images.githubusercontent.com/89505134/133542270-e9343c27-03b5-4824-8219-dd11a82f5470.png)
![Screenshot from 2021-09-16 08-08-11](https://user-images.githubusercontent.com/89505134/133542276-12bb5d4d-99c4-4c7b-8c0f-aded517ef3b2.png)
![Screenshot from 2021-09-16 08-08-16](https://user-images.githubusercontent.com/89505134/133542279-4df358ab-d180-4146-9a36-2065fcc30154.png)
![Screenshot from 2021-09-16 08-08-26](https://user-images.githubusercontent.com/89505134/133542281-05b51d63-66a5-431d-ad23-011ce1c9f5a8.png)
![Screenshot from 2021-09-16 08-08-31](https://user-images.githubusercontent.com/89505134/133542284-44f36305-4ecd-44e0-9a02-bc471d95a6b2.png)
![Screenshot from 2021-09-16 08-08-59](https://user-images.githubusercontent.com/89505134/133542287-70b16480-8074-4cb0-b1f3-e3847e2a63b3.png)
![Screenshot from 2021-09-16 08-09-05](https://user-images.githubusercontent.com/89505134/133542289-480da21c-0deb-4a15-8111-81570258ceef.png)
![Screenshot from 2021-09-16 08-09-13](https://user-images.githubusercontent.com/89505134/133542290-f5ee287a-7433-4afb-8963-0456965300a8.png)
![Screenshot from 2021-09-16 08-09-19](https://user-images.githubusercontent.com/89505134/133542292-3ec06c33-da71-4e21-9996-759b04d08651.png)
